code/game/gamemodes/wizard/artefact.dm

```diff
@@ -136,4 +136,66 @@
 			user.on_moved.Remove(event_key)
 			event_key = null
 			user.alpha = initial(user.alpha)
-			user.alphas.Remove(CLOAKINGCLOAK)
\ No newline at end of file
+			user.alphas.Remove(CLOAKINGCLOAK)
+
+
+/obj/item/weapon/glow_orb
+	name = "inert stone"
+	desc = "A peculiar fist-sized stone which hums with dormant energy."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "glow_stone_dormant"
+	w_class = W_CLASS_TINY
+	force = 0
+	var/prime_time = 2 SECONDS
+	var/crit_failure = 0
+	var/activating = 0
+
+/obj/item/weapon/glow_orb/attack_self(mob/user)
+	if(crit_failure)
+		to_chat(user, "<span class = 'warning'>\The [src] is vibrating erratically!</span>")
+		return
+	if(activating)
+		to_chat(user, "<span class = 'warning'>\The [src] hums with energy as it begins to glow brighter.</span>")
+		return
+	if(iswizard(user) || isapprentice(user))
+		to_chat(user, "<span class = 'notice'>You prime the glow-stone, it will transform in [prime_time/10] seconds.</span>")
+		activate()
+		return
+	else
+		if (clumsy_check(user) && prob(50))
+			to_chat(user, "<span class = 'notice'>Ooh, shiny</span>")
+			failure()
+			return
+		else if(prob(65))
+			to_chat(user, "<span class = 'notice'>You find what appears to be an on button, and press it.</span>")
+			activate()
+		else
+			if(prob(5))
+				visible_message("<span class = 'warning'>\The [src] ticks [pick("ominously","forebodingly", "harshly")]</span>")
+				if(prob(50))
```

Comment:
There's a 3.25% chance of them even getting this far, I think you can remove this one.